google/cloud/documentai_toolbox/wrappers/entity.py

```diff
@@ -59,32 +75,36 @@ def __post_init__(self):
         ):
             self.normalized_text = self.documentai_entity.normalized_value.text
 
-        if self.documentai_entity.page_anchor.page_refs:
-            self.start_page = int(self.documentai_entity.page_anchor.page_refs[0].page)
-            self.end_page = int(self.documentai_entity.page_anchor.page_refs[-1].page)
+        page_refs = self.documentai_entity.page_anchor.page_refs
+        if page_refs:
+            self.start_page = int(page_refs[0].page) + page_offset
+            self.end_page = int(page_refs[-1].page) + page_offset
```

Comment:
We should add a unit test for these lines of code above.

Reply:
Done